content/blog/getting-started-with-hpe-greenlake-api-for-data-services.md

```diff
@@ -0,0 +1,242 @@
+        {
+            "message": "Task is running",
+            "timestampAt": "2024-03-24T00:13:52.002675372Z"
+        },
+        {
+            "message": "Preparing parameters",
+            "timestampAt": "2024-03-24T00:13:53.368619324Z"
+        },
+        {
+            "message": "Starting virtual machine deployment",
+            "timestampAt": "2024-03-24T00:13:53.558043002Z"
+        },
+        {
+            "message": "Applying protection policy",
+            "timestampAt": "2024-03-24T00:15:49.598976645Z"
+        },
+        {
+            "message": "Virtual machine provisioning completed and initiated a task for applying backup policy.",
+            "timestampAt": "2024-03-24T00:15:49.906721852Z"
+        },
+        {
+            "message": "Task succeeded",
+            "timestampAt": "2024-03-24T00:15:49.906727488Z"
+        }
+    ],
+    "name": "Provisioning virtual machine 0-RRD-API-Deploy-4",
+    "progressPercent": 100,
+    "services": [
+        "private-cloud-business-edition"
+    ],
+    "startedAt": "2024-03-24T00:13:52.002663421Z",
+    "state": "SUCCEEDED",
+    "suggestedPollingIntervalSeconds": 30,
+    "type": "task",
+    "updatedAt": "2024-03-24T00:15:49.955699371Z"
+}
+```
+
+The above f﻿igure indicated that the second poll of the VM provisioning REST API **task Id** indicated that the creation of the virtual machines on-premises had completed successfully (**progressPercent: 100, state: SUCCEEDED**). User then can discover a VM that is named “0-RRD-API-Deploy-4” available at the VMware cluster where this provisioning was executed. 
+
+# Summary
+
+This blog post introduces you to the new set of HPE Greenlake REST APIs for data services, to support resources such as: *async-operations, dual-auth-operations, issues, secrets, software-releases, storage locations, and tags.* This set of APIs will evolve throughout the future toward long term supported version of the APIs. 
+
+This March 2024 announcement introduces revision v1beta1 of the API; which is documented at HPE GreenLake Developer [website](https://developer.greenlake.hpe.com/docs/greenlake/services/data-services/public/) using the interactive reference documentation based on OpenAPI 3.1 standard. In this post, I also introduced methods to exercise the API directly from the API reference documentation page using the access token obtained from HPE GreenLake. Lastly, I provided a list of tips on using these HPE GreenLake REST API for specific use cases. 
+
+Please don’t hesitate to explore this new set of HPE GreenLake APIs for data services and see how you can improve your agility in managing your data. If you have any questions on any of these APIs, or if you are interested to share your feedback and use cases on this set of API;  please join the HPE Developer Slack [Workspace](https://developer.hpe.com/slack-signup), and start a discussion in our [*\#hpe-greenlake-data-services*](https://hpedev.slack.com/archives/C02D6H623JP) slack channel.
```

Comment:
Change "to share your feedback" to "in sharing your feedback".
Change "on this set of API" to "on this set of APIs".
Change the semi-colon after "API" to a comma.
Remove the comma after "Slack Workspace"
Capitalize the "S" in "Slack channel" at the very end.